src/config.rs

```diff
@@ -78,6 +78,7 @@ pub(crate) struct RelabelConfig {
 #[derive(PartialEq, Eq, Debug, serde::Deserialize)]
 pub(crate) struct PrioritizeConfig {
     pub(crate) label: String,
+    pub(crate) prioritize_on: Vec<String>,
```

Comment:
I think we'll want to mark this with `#[serde(default)]` as well to avoid figuring out whether the rust-lang/rust PR needs to land first or this one.

Reply:
If any repository other than `rust-lang/rust` needs to use this feature, they might have different needs and not use the `prioritize_on` field, so I also think we should mark it with `#[serde(default)]`

Reply:
Done.